map/local_data.html

```diff
@@ -65,6 +85,24 @@ <h1>BCU Labs - Data Team</h1>
             type: 'geojson',
             data: 'plots/LTS.json'
         });
+        const colors = [
+            'black',
+            // Colors based on 5 equally spaced from 'turbo' colormap
+            '#28BCEB',
+            '#A4FC3C',
+            '#FB7E21',
+            '#7A0403',
+            'white'
+        ]
+        const no_data = 'no data'
+        const LTS_names = [
+            no_data, // black
+            'Level of Traffic Stress 1',
```

Comment:
These labels seem too technical to be user-facing but I wasn't sure what to put instead.

Reply:
I am working on learning Angular per @tschieggm suggestion for a front end. My thought was to make the website a new repo that would have everything public facing there. The thought with it being a different repo than this one is to eventually add the crash, 331, etc. maps as layers on top of the stress map. With that in mind, I think keeping a testing map in this repo with a simple and technical legend, etc. is reasonable for us to do technical development. I am open to other thoughts on how to best handle everything.

Reply:
As for less technical wording, I think Adam has come up with some great concepts for the UI on slack. I recommend looking at those.